common/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="TypographyEllipsis">
+    <string name="account_ledger_migration_app">Migration</string>
+    <string name="account_ledger_migration_generic">Generic</string>
+
+    <string name="account_ledger_generic_import_start_step_1">Make sure <font color='#FFFFFF'>Generic app is installed</font> to your Ledger device using Ledger Live app</string>
+    <string name="account_ledger_generic_import_start_step_2"><font color='#FFFFFF'>Open the Generic app</font> on your Ledger device</string>
```

Comment:
Yeah this is the old way but this is basically the mostly copy of the legacy flow with some logic changes. I also thought aboure SpannableFormatter but decided to leave this refactoring for now since we want to implement Ledger as fast as possible